_topic_maps/_topic_map_rosa.yml

```diff
@@ -1606,6 +1601,124 @@ Topics:
   - Name: Glossary
     File: logging-common-terms
 ---
+Name: Backup and restore
+Dir: backup_and_restore
+Distros: openshift-rosa
+Topics:
+- Name: Overview of backup and restore operations
+  File: index
+- Name: Shutting down a cluster gracefully
+  File: graceful-cluster-shutdown
```

Comment:
This does not works for ROSA/OSD, customer's can hibernate/resume their cluster after getting a support exception approved. Ref: https://access.redhat.com/articles/7012966